animations/test/misc/expand_card_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/expand_card.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createExpandCardScreen() => MaterialApp(
+      home: ExpandCardDemo(),
+    );
+
+void main() {
+  testWidgets('size of ExpandCard changes on tap', (tester) async {
```

Comment:
This is a good idea for a test, but it's reaching farther into the details of ExpandCard than it needs to. For example, it's coupled to the fact that ExpandCard is using an AnimatedContainer widget internally and the exact sizes of that widget (128, 256).

Consider instead merely checking that the ExpandCard itself gets bigger. You could:

* Create the screen
* Find the ExpandCard widget and record its size
* Tap directly on it
* pumpAndSettle (no duration necessary, I believe)
* Check the size again and verify just that it's bigger than the first value.

This would decouple your test from the specific size and the exact widgets being used *inside* ExpandCard. If you simulate the tap on ExpandCard itself, you'd also be able to avoid using hardcoded screen coordinates. That'd be a good thing, since they would change if the test were run on different devices.

Reply:
Makes sense, I was focusing on the implementation rather than `ExpandCard` itself. I'll do the required changes.